README.md

```diff
@@ -238,6 +239,12 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Guice](https://github.com/google/guice) - Lightweight but powerful framework that completes Dagger.
 * [HK2](https://hk2.java.net) - Light-weight and dynamic dependency injection framework.
 
+## Deployment
```

Comment:
Is a new category with a single entry really necessary? There is a `Distribution` section which might be suited.